build-tools/packages/build-cli/.eslintrc.cjs

```diff
@@ -16,6 +16,7 @@ module.exports = {
     ],
     rules: {
         "@typescript-eslint/no-unused-vars": "warn",
+        "unused-imports/no-unused-imports": "warn",
```

Comment:
Do we really need this? Seems like it would be better to just remove the violators and re-add in future changes as needed IMO

Reply:
Ehhh, I'm reluctant to make this an error. I really hate having to clean these things up and re-add them all the time; I'd rather do a single "we're done" pass and enable the rule then. It gives me Go PTSD, because Go won't even compile code with unused variables, and I absolutely hate(d) it.